drivers/sensor/vl53l1x/vl53l1_platform_user_config.h

```diff
@@ -1,29 +1,8 @@
-/*******************************************************************************
-COPIED FROM HERE: https://github.com/bitcraze/crazyflie-firmware-experimental/blob/master/src/lib/vl53l1/vl53l1_platform_user_config.h
-Copyright (C) 2015, STMicroelectronics International N.V.
-All rights reserved.
-Redistribution and use in source and binary forms, with or without
-modification, are permitted provided that the following conditions are met:
-	* Redistributions of source code must retain the above copyright
-	  notice, this list of conditions and the following disclaimer.
```

Comment:
Deleting these lines is a clear violation of the copyright.

Reply:
Sorry, I'm going to check the references PR and make change in order see if its possible to integrate this driver. Thanks.